CHANGES.rst

```diff
@@ -1,7 +1,17 @@
-2.1.1 (unreleased)
+Unreleased
 ------------------
 
-- Nothing changed yet.
+- Project renamed back to ``flake8-commas``
```

Comment:
My current thinking is that this would be v4 of `flake8-commas` (and `flake8-commas` would then simply not have a v3). Open to other suggestions.

Reply:
Have updated https://github.com/PyCQA/flake8-commas/pull/70/commits/632f6a8a54d090790cf235acf120a18795b148eb to take this approach.